packages/app/src/app/pages/Sandbox/Editor/Workspace/Files/DirectoryEntry/Entry/EditIcons/index.js

```diff
@@ -36,45 +35,43 @@ function EditIcons({
 
   return (
     <div className={className}>
-      {(hovering || (window.__isTouch && active) || forceShow) && (
-        <Container>
```

Comment:
Also, none of this code has changed. Only the containing `{(hovering || .......... && ( ... )}` thing was removed.